data/reusables/enterprise_migrations/ready-to-import-migrations.md

```diff
@@ -1 +1 @@
-1. To import the archived migration data to a {% data variables.product.prodname_ghe_server %} instance, see "[Importing migration data to {% data variables.product.prodname_ghe_server %}](/enterprise/admin/guides/migrations/importing-migration-data-to-github-enterprise-server/)".
+1. To import the archived migration data to a {% data variables.product.prodname_ghe_server %} instance, see "[Preparing to migrate data to your enterprise](/enterprise/admin/user-management/preparing-to-migrate-data-to-your-enterprise)".
```

Comment:
Hi @ethanpalm 👋🏻 

Thank you for your contribution 😄 

Looking at [Exporting migration data from GitHub.com - GitHub Docs](https://docs-2930--bugfix-ghes-migrati.herokuapp.com/en/enterprise-server@2.22/admin/user-management/exporting-migration-data-from-githubcom#generating-a-migration-archive), I can see that your PR nicely fixes the reported issue.

I think that we might want to go slightly further and also update the text in the step to emphasize the need to prepare the data before importing it. How about:

```suggestion
1. To prepare the archived migration data for import into a {% data variables.product.prodname_ghe_server %} instance, see "[Preparing to migrate data to your enterprise](/enterprise/admin/user-management/preparing-to-migrate-data-to-your-enterprise)".
```